lcm-cmake/lcmUtilities.cmake

```diff
@@ -270,6 +270,8 @@ function(lcm_wrap_types)
   foreach(_lcmtype ${_UNPARSED_ARGUMENTS})
     set(_package "")
     set(_outputs "")
+    unset(_package_dir)
```

Comment:
I think `set(X "")` would be more consistent.

Reply:
That's a fair point.
Updated